apps/files/utils.py

```diff
@@ -208,6 +212,8 @@ def find_type(self):
         # https://developer.mozilla.org/en-US/Add-ons/Install_Manifests#type
         self.package_type = self.find('type')
         if self.package_type and self.package_type in self.TYPES:
+            # If it's an experiment, we need to store that for later.
+            self.is_experiment = (self.package_type == '128')
```

Comment:
I don't think those parens are needed. Is that a flake8 thing or does it just look weird without?